### DISCUSSION TITLE
Transfer domain associated with deleted team

### DISCUSSION THREAD

**Discussion by cadrianxyz** [vercel/community#3828] [Help] [answered]
### Summary

I setup a new team for my hosted site (`ceco-ai`), and when trying to link my domain (`app.ceco.ai`), realize it was linked to another person's Vercel account who has since been inactive on the project. He tried deleting the team on his end (with the attached domain), and after waiting 5 business days it still says that the "Domain was added to a different project".

I'd like help transferring the domain to my new account/team. Thank you!

This issue is similar to #3769.

### Steps to Reproduce

1. Create team
2. Create and deploy app associated with team
3. Purchase domain through team
4. Assign domain to deployed app in step 2
5. Transfer app to personal account
6. Delete team

**Comment by amyegan** [comment]
Hey @ad2969. It sounds like your friend currently has control of the domain. They can transfer the domain to you by following these steps: https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team

**Comment by cadrianxyz** [comment]
> Hey @ad2969. It sounds like your friend currently has control of the domain. They can transfer the domain to you by following these steps: https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team

Yes, that is the case I explained. He has already deleted the team (with the attached domain), so "transferring" right now does not look like an option.

**Reply by amyegan** [reply, marked as answer]
The team and project still seem to exist. Please reach out to your friend who owns the domain.

**Reply by cadrianxyz** [reply]
Thank you for checking this! This info is helpful
